packages/suite/src/views/settings/SettingsDevice/DisplayRotation.tsx

```diff
@@ -21,12 +22,20 @@ interface DisplayRotationProps {
 export const DisplayRotation = ({ isDeviceLocked }: DisplayRotationProps) => {
     const dispatch = useDispatch();
     const { device } = useDevice();
-    const currentRotation = device?.features?.display_rotation;
 
-    if (typeof currentRotation !== 'number') {
+    // features.display_rotation cannot be used to determine support because can be defined for devices not supporting rotation (e.g. T2B1).
```

Comment:
I assume we don't have this in `Capabilities` right? Is there issue or something where this is covered in FW?